roles/control_center/tasks/main.yml

```diff
@@ -149,7 +149,7 @@
     kerberos_keytab_path: "{{control_center_kerberos_keytab_path}}"
     kerberos_keytab_destination_path: "{{control_center_keytab_path}}"
     kerberos_handler: "restart control center"
-  when: kafka_broker_listeners[control_center_kafka_listener_name]['sasl_protocol'] | default(sasl_protocol) | confluent.platform.normalize_sasl_protocol == 'GSSAPI'
+  when: "'GSSAPI' in (kafka_broker_listeners[control_center_kafka_listener_name]['sasl_protocol'] | default(sasl_protocol) | confluent.platform.normalize_sasl_protocol)"
```

Comment:
what will happen when control_center_kafka_listener_name as multiple protocols defined?

Reply:
This just adds kerberos support to the component.